Gemfile

```diff
@@ -1,9 +1,7 @@
-ruby "~> 2.3.0"

```

Comment:
The reason of removing this is?

Reply:
Because when I direct run commands in directory, while "bundle install", it shows error "Your Ruby version is 2.6.5, but your Gemfile specified ~> 2.3.0". And the removal of it clean the error.